lib/kitchen/transport/winrm.rb

```diff
@@ -87,6 +87,10 @@ def close
         def execute(command)
           return if command.nil?
           logger.debug("[WinRM] #{self} (#{command})")
+
+          if command.length > MAX_COMMAND_SIZE
+            command = run_from_file_command(command)
```

Comment:
Also, feel free to disagree with me and leave this the way it is - my comment is pretty nitpicky.